### PR TITLE
 Apply downstream patches for gnome-user-docs 43

### PR DESCRIPTION
**Pull request by dylanmccall**
https://phabricator.endlessm.com/T35074

-----

Replaces #47.

### CODE REVIEW

**Comment by dylanmccall** [comment]
It was bugging me, so I went ahead and dropped [[ https://github.com/endlessm/gnome-user-docs/pull/50/commits/744c1bc400d2ffa4430ba845e1f85fe5c94a5801 | our additions to touchscreen-gestures.page ]], and adjusted the screenshots in shell-exit.page and shell-introduction.page so we have the two different screenshots of the system menu (with and without the shutdown menu expanded), as in the upstream docs.

**Review by wjt — APPROVED** [review]
I amended two commits to fix some minor typos.

Nice work!

**Comment by wjt** [comment]
I force-pushed this branch to master.

File: gnome-help/C/shell-apps-open.page [hunk 1]
```diff
@@ -69,9 +74,10 @@ icon to choose a window to focus or get new window. Mention middle-click?</p>
     </item>
     <item>
       <p>Click the grid button (which has nine dots) in the dash.
-      You will see the first page of all installed applications. To see more
-      applications, press the dots at the bottom, above the dash, to view other
-      applications. Press on the application to start it.</p>
+      You will see the first page of the desktop. To see more applications,
+      press the dots at the bottom, above the dash, to view other applications
+      or move your mouse cursor to the right-most side of the screen and select
+      the arrow button that appears. Press on the application to start it.</p>
```
[the remark this answers is not in the document]
Comment (dylanmccall) [reply gnome-help/C/shell-apps-open.page:80]: > Personally I wouldn't even mention that the dots are clickable.

TIL the dots are clickable :b

**Pull request closed without merging**